package.json

```diff
@@ -50,14 +51,15 @@
     "@formatjs/cli": "^4.2.20",
     "@testing-library/jest-dom": "^5.11.1",
     "@testing-library/react": "^11.0.4",
-    "babel-jest": "^26.4.2",
+    "babel-jest": "^29.5.0",
```

Comment:
Do tests work without `babel-jest`, `jest`, `jest-environment-jsdom`? All these deps are already listed in `jest-config-stripes`.

Reply:
Unfortunately, tests not working without those deps locally

Reply:
Hey @przemyslawturek, are you running tests in a standalone module or as a part of platform? I've just installed this module separately from platform, without jest dependencies and tests are working correctly.
If it's a part of platform - there could be failing tests due to different jest versions in different modules.

Unfortunately test-related dependencies haven't been consistently kept in sync, and removing them from `devDependencies` and relying on `jest-config-stripes` to provide them is going to fix this problem

Reply:
Also, could you provide what error you are seeing in tests?

Reply:
This may be a local issue, please push a commit without those deps.

Reply:
Yes it seems this problem is only on standalone version running separately outside platform. Sorry guys for turning you into the wrong path, my bad